### PR TITLE
CORTX-28629: Handle multiple index mapping to rgw service instance.

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
# Problem Statement
- Handle multiple index mapping to rgw service instance.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/const.py [hunk 1]
```diff
@@ -26,9 +26,15 @@
 RGW_CONF_FILE = f'cortx_{COMPONENT_NAME}.conf'
 # e.g. RGW_CONFI_FILE path will be cortx_rgw.conf
 REQUIRED_RPMS = ['cortx-hare', 'cortx-py-utils', 'ceph-radosgw']
+RGW_ADMIN_PARAMETERS = {'MOTR_ADMIN_FID':'motr admin fid', 'MOTR_ADMIN_ENDPOINT':'motr admin endpoint'}
```
Comment (rahul-tripathi-git) [inline src/const.py:29]: Can we remove RGW from this variable name?

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

File: conf/cortx_rgw.conf [hunk 2]
```diff
@@ -3,13 +3,21 @@
...
+
+[client.radosgw-admin]
+    motr profile fid =
+    motr ha endpoint =
+    motr my endpoint =
+    motr my fid =
+    motr admin fid =
+    motr admin endpoint =
+    log file =
+
```
Comment (ujjwalpl) [inline conf/cortx_rgw.conf:15]: Remove extra line or be consistent in spaces between the stanzas.

File: src/setup/rgw_setup [hunk 3]
```diff
@@ -38,6 +38,7 @@ class SetupCmdBase(Cmd):
     def add_args(parser):
         parser.add_argument('--services', default='services', help='services')
         parser.add_argument('-c', '--config', default='config_url', help='config')
+        parser.add_argument('--index', default='1', help='index')
```
Comment (ujjwalpl) [inline src/setup/rgw_setup:41]: parser.add_argument('--index', default='1', help='service sequence index')

File: src/setup/rgw_service [hunk 4]
```diff
@@ -14,10 +14,12 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+index=$1
+
 function start_rgw {
   machine_id=$(cat /etc/machine-id)
   # TODO: To use log dir from config
-  exec sudo /usr/bin/radosgw -f --name client.rgw -c /etc/cortx/rgw/"$machine_id"/cortx_rgw.conf --no-mon-config &> /var/log/ceph/rgw.log &
+  exec sudo /usr/bin/radosgw -f --name client.rgw-"$index" -c /etc/cortx/rgw/"$machine_id"/cortx_rgw.conf --no-mon-config &> /var/log/ceph/rgw.log &
```
Comment (ujjwalpl) [inline src/setup/rgw_service:22]: We should not use hard coded path. This should the config path

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: conf/cortx_rgw.conf (hunk 2 above)
Comment (shriya-deshmukh) [reply conf/cortx_rgw.conf:15]: done.

File: src/setup/rgw_service (hunk 4 above)
Comment (shriya-deshmukh) [reply src/setup/rgw_service]: Removed hardcoded path.

File: src/setup/rgw_setup (hunk 3 above)
Comment (shriya-deshmukh) [reply src/setup/rgw_setup]: done.

**Review by ujjwalpl — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
looks good

**Pull request merged**